### PR TITLE
User settings

### PR DESCRIPTION
**Pull request by Rostyk1404**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPYTTTN-29)

## Code reviewers

- [ ] @antooa
- [ ] @HalynaDom
- [ ] @OlexaNdrus
- [ ] @pinkfoxxx
- [ ] @Rocckk
- [ ] @Sent1nelDNoir
- [ ] @Svviat
- [ ] @ValentynLiashenko

### Second Level Review

- [ ] @Blackleaves
- [ ] @yevgenbehen
- [ ] @lhalam

## Summary of issue

User settings

### CODE REVIEW

File: iBudget/templates/emails/change_password_link.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<h1>Hello
+    <i>
```
Comment (Blackleaves) [inline iBudget/templates/emails/change_password_link.html]: It is better to use less indentation in email templates. Sometimes these indentation will appear in real email.

File: iBudget/templates/emails/update_password.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<h2>Your password has been changed
+    <i>
```
Comment (Blackleaves) [inline iBudget/templates/emails/update_password.html]: Same as previous.

File: iBudget/static/src/components/Login.vue [hunk 3]
```diff
@@ -1,136 +1,175 @@
...
-                <hr/>
-                <b-button type="submit" @click="registration">Register</b-button>
-               <br/>
-               <br/>
-               <b-link @click="showLogin">Already registered?</b-link>
-             </form>
-           </div>
-      </div>
-  </div>
+    <div class="content">
```
Comment (Blackleaves) [inline iBudget/static/src/components/Login.vue]: Make a mark for yourself that all this `<br>` should be replaced with proper styles in future.

File: iBudget/templates/emails/update_password.html (hunk 2 above)
Comment (Rostyk1404) [reply iBudget/templates/emails/update_password.html]: I will do as u said in the future.Thank u

File: iBudget/static/src/components/Login.vue (hunk 3 above)
Comment (Rostyk1404) [reply iBudget/static/src/components/Login.vue]: I will do as u said in the future.Thank u

**Pull request merged**